mqtt-cs/src/main/java/org/apache/rocketmq/mqtt/cs/config/ConnectConf.java

```diff
@@ -61,6 +61,7 @@ public class ConnectConf {
     private boolean enableRetain = false;
     private boolean enableSubscriptionIdentifier = false;
     private int topicAliasMaximum = 10;
+    private int serverReceiveMaximum = 65535;
```

Comment:
why set to 65535？

Reply:
According to MQTT protocol, the initial Receive Maximum is 65535.